WS2012R2/lisa/xml/STOR_VHDX.xml

```diff
@@ -114,6 +116,7 @@
                 <param>DefaultSize=64TB</param>
                 <!-- <param>Path=G:\VHDs</param> -->
                 <param>ControllerType=SCSI</param>
+                <param>SCSI=0,0,Dynamic,512,64TB</param>
```

Comment:
this param is used to Count the number of SCSI= and IDE= entries in constants in /STOR_Large_Disk.sh